cpp/example_code/dynamodb/dynamodb_partiql_scenario.cpp

```diff
@@ -0,0 +1,568 @@
+                  << TITLE_KEY << "=? and " << YEAR_KEY << "=?";
+
+        request.SetStatement(sqlStream.str());
+
+        Aws::Vector<Aws::DynamoDB::Model::AttributeValue> attributes;
+        attributes.push_back(Aws::DynamoDB::Model::AttributeValue().SetS(title));
+        attributes.push_back(Aws::DynamoDB::Model::AttributeValue().SetN(year));
+        request.SetParameters(attributes);
+
+        Aws::DynamoDB::Model::ExecuteStatementOutcome outcome = dynamoClient.ExecuteStatement(
+                request);
+        if (!outcome.IsSuccess()) {
+            std::cerr << "Failed to retrieve the movie information: "
+                      << outcome.GetError().GetMessage() << std::endl;
+            return false;
+        }
+        else {
+            const Aws::DynamoDB::Model::ExecuteStatementResult &result = outcome.GetResult();
+
+            const Aws::Vector<Aws::Map<Aws::String, Aws::DynamoDB::Model::AttributeValue>> &items = result.GetItems();
+
+            if (items.size() == 1) {
+                printMovieInfo(items[0]);
+            }
+            else {
+                std::cerr << "Error: " << items.size() << " movies were retrieved. "
+                          << " There should be only one movie." << std::endl;
+            }
+        }
+    }
+
+    std::cout << "Deleting the movie" << std::endl;
+
+    // Delete the movie using a "Delete" statement. (ExecuteStatement)
+    {
+        Aws::DynamoDB::Model::ExecuteStatementRequest request;
+        std::stringstream sqlStream;
+        sqlStream << "DELETE FROM  \"" << MOVIE_TABLE_NAME << "\" WHERE "
+                  << TITLE_KEY << "=? and " << YEAR_KEY << "=?";
+
+        request.SetStatement(sqlStream.str());
+
+        Aws::Vector<Aws::DynamoDB::Model::AttributeValue> attributes;
+        attributes.push_back(Aws::DynamoDB::Model::AttributeValue().SetS(title));
+        attributes.push_back(Aws::DynamoDB::Model::AttributeValue().SetN(year));
+        request.SetParameters(attributes);
+
+        Aws::DynamoDB::Model::ExecuteStatementOutcome outcome = dynamoClient.ExecuteStatement(
+                request);
+        if (!outcome.IsSuccess()) {
+            std::cerr << "Failed to delete the movie: "
+                      << outcome.GetError().GetMessage() << std::endl;
+            return false;
+        }
+    }
+
+    std::cout << "Movie successfully deleted." << std::endl;
+    return true;
+}
+// snippet-end:[cpp.example_code.dynamodb.Scenario_PartiQL_Single]
+
+// snippet-start:[cpp.example_code.dynamodb.Scenario_PartiQL_Batch]
+//! Scenario to modify and query a DynamoDB table using PartiQL batch statements.
+/*!
+  \sa partiqlBatchExecuteScenario()
+  \param clientConfiguration: Aws client configuration.
+  \return bool: Function succeeded.
+ */
+bool AwsDoc::DynamoDB::partiqlBatchExecuteScenario(
+        const Aws::Client::ClientConfiguration &clientConfiguration) {
+    std::cout << "Now we will work with batches of movies" << std::endl;
```

Comment:
Missing punctuation?

Reply:
This was removed when I broke up the scenarios.  It is no longer a segue between the two.